src/generic_core/remote-connection.ts

```diff
@@ -153,7 +153,8 @@ var generateProxyingSessionId_ = (): string => {
       this.rtcToNet_ = new rtc_to_net.RtcToNet(this.userId_);
       this.rtcToNet_.start({
         allowNonUnicast: globals.settings.allowNonUnicast,
-        reproxy: globals.settings.reproxy
+        reproxy: globals.settings.reproxy,
+        userId: this.userId_
```

Comment:
Do you need this? Seems like `RtcToNet` already knows the `userId` since its constructor accepts it as an argument.